app/assets/javascripts/miq_c3.js

```diff
@@ -27,8 +27,8 @@ function load_c3_chart(data, chart_id, height) {
   generate_args.data.onclick = function(data, _i) {
     var index = _.findIndex(generate_args.data.columns, function(col) { return col[0] == data.id; });
     // when not Pie/Donut chart, first column doesn't contain actual data.
-    var seriesIndex = _.contains(['Pie', 'Donut'], generate_args.miqChart) ? index : index - 1;
-    var pointIndex = _.contains(['Pie', 'Donut'], generate_args.miqChart) ? index : data.index;
+    var seriesIndex = ['Pie', 'Donut'].includes(generate_args.miqChart) ? index : index - 1;
+    var pointIndex = ['Pie', 'Donut'].includes(generate_args.miqChart) ? index : data.index;
```

Comment:
I hope we got a shim for [this](https://caniuse.com/#search=includes)...

Reply:
es6-shim :)